src/types.rs

```diff
@@ -231,7 +242,7 @@ key!(Bytes8, 8);
 key!(Bytes32, 32);
 key!(Salt, 32);
 
-key_no_default!(Bytes64, 64);
+key_no_default_large!(Bytes64, 64);
```

Comment:
@vlopes11 why was this called `key_no_default`? It seems like it already has a default impl.

Reply:
@Voxelot likely a placeholder, any recommendations on what to rename it?

Reply:
@Voxelot it was previously called `key_no_default` because it did all the same stuff _except_ `#[derive(Default)]`. Because it was only used in one place, and this PR furthers the difference between this macro and `key!`, I suggested the name change to reflect the fact that it uses `BigArray`.

Reply:
> I suggested the name change to reflect the fact that it uses `BigArray`.

Something like, `key_big_array!` ?